config_d/registers

```diff
@@ -55,6 +55,10 @@ BLOCKS_COMPAT_VERSION = 0
     # frequency of 125 MHz will be reported and used.
     NOMINAL_CLOCK           24
 
+    # Hardware timestamps
+    PCAP_TS_SEC             25
+    PCAP_TS_NSEC            26
+
```

Comment:
Could we put these with the other PCAP ones in `10` and `11`?

Reply:
Done